Framework/PythonInterface/plugins/algorithms/CylinderPaalmanPingsCorrection2.py

```diff
@@ -405,10 +405,10 @@ def _set_material(self, ws_name, chemical_formula, density_type, density):
             mat = builder.setFormula(chemical_formula).setMassDensity(density).build()
```

Comment:
I think we can simplify this a bit now can't we? The `builder` variable is no longer needed as the number density will be calculated on the fly? So basically the `if` just needs 1 line each for whether the property is `SampleMassDensity` or `SampleNumberDensity`?